usr/share/rear/prep/BACULA/default/500_check_BACULA_bconsole_results.sh

```diff
@@ -11,14 +11,12 @@ fi
 #
 # See if we can ping the director
 #
-# is the director server present? Fetch from /etc/bacula/bconsole.conf file
-BACULA_DIRECTOR=$(grep -i address /etc/bacula/bconsole.conf | awk '{ print $3 }')
-[ "${BACULA_DIRECTOR}" ]
-StopIfError "Director not defined in /etc/bacula/bconsole.conf"
+# is the director server present? Fetch from $BACULA_CONF_DIR/bconsole.conf file
+BACULA_DIRECTOR=$(grep -i address $BACULA_CONF_DIR/bconsole.conf | awk '{ print $3 }')
+[ "${BACULA_DIRECTOR}" ] || Error "Director not defined in $BACULA_CONF_DIR/bconsole.conf"
 
 if test "$PING"; then
-	ping -c 2 -q  $BACULA_DIRECTOR >/dev/null
-	StopIfError "Backup host [$BACULA_DIRECTOR] not reachable."
+	ping -c 2 -q  $BACULA_DIRECTOR >/dev/null || Error "Backup host [$BACULA_DIRECTOR] not reachable."
```

Comment:
Only FYI:
In general 'ping' is a poor test to find out if a particular service on a particular host
is accessible because sometimes a host does not respond to a 'ping' but its
service is well accessible.
Example (test if www.suse.com is accessible via HTTP):
```
# time ping -c 1 -w 10 &>/dev/null www.suse.com && echo Y || echo N
real    0m10.050s
user    0m0.005s
sys     0m0.000s
N

# time netcat -z -w 10 www.suse.com 80 && echo Y || echo N
real    0m0.034s
user    0m0.002s
sys     0m0.000s
Y
```